src/components/AssetManagementPanel/AssetManagementForm/FormVariants/TransferHolderForm/TransferHolderForm.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from "react";
+import { ButtonSolidOrangeWhite, ButtonSolidWhiteGrey } from "../../../../UI/Button";
+import { AssetInformationPanel } from "../../../AssetInformationPanel";
+import { AssetManagementActions } from "../../../AssetManagementActions";
+import { AssetManagementTitle } from "../../AssetManagementTitle";
+import { EditableAssetTitle } from "./../EditableAssetTitle";
+
+interface TransferHolderProps {
+  formAction: AssetManagementActions;
+  onSetFormAction: (nextFormAction: AssetManagementActions) => void;
+  tokenId: string;
+  tokenRegistryAddress: string;
+  beneficiary?: string;
+  holder?: string;
+  handleTransfer: (newHolder: string) => void;
+  holderState: string;
+}
+
+export const TransferHolderForm = ({
+  formAction,
+  onSetFormAction,
+  tokenId,
+  tokenRegistryAddress,
+  beneficiary,
+  holder,
+  handleTransfer,
+  holderState,
+}: TransferHolderProps) => {
+  const [newHolder, setNewHolder] = React.useState<string>("");
```

Comment:
```suggestion
  const [newHolder, setNewHolder] = useState("");
```

useState will infer from default value. Also just destructure useState from react in the import